components/widget/widget_table.lua

```diff
@@ -11,6 +11,12 @@ local Class = require('Module:Class')
 local Widget = require('Module:Infobox/Widget')
 local WidgetFactory = require('Module:Infobox/Widget/Factory')
 
+---@class WidgetTable
+---@field rows (WidgetTableRow|nil)[]
+---@field classes string[]
+---@field css {[string]: string|number|nil}[]
+---@field columns integer?
+---@field injector function?
```

Comment:
```suggestion
---@class WidgetTable:Widget
---@field rows (WidgetTableRow|nil)[]
---@field classes string[]
---@field css {[string]: string|number|nil}[]
---@field columns integer?
```
injector field belongs to the parent, Widget

Reply:
that adds warning "Undefined field `injector`"

Reply:
Yeah, Infobox/Widget need to be set with `---@class Widget` and the injector field

Reply:
yeah i thought so too, but still get the warning with that

![Screenshot 2023-05-25 08 07 24](https://github.com/Liquipedia/Lua-Modules/assets/75081997/b62a5ba8-ae39-43a7-82e5-54eaa5816449)